src/test/java/org/folio/logintest/CookieTest.java

```diff
@@ -0,0 +1,44 @@
+package org.folio.logintest;
+
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThrows;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import org.junit.Test;
+
+import org.folio.rest.impl.LoginAPI;
+import org.folio.util.TokenCookieParser;
+
+public class CookieTest {
```

Comment:
When you rename and move org.folio.logintest.CookieTest to org.folio.util.TokenCookieParserTest it's more easy for me to find the related unit tests.

Reply:
Sure